### ISSUE TITLE
Float the mute/unmute, camera and other actions when any element in fullscreen

### ISSUE THREAD

**Issue by ravi-ct** [bigbluebutton/bigbluebutton#9947]
**Is your feature request related to a problem? Please describe.**
When users are watching either a webcam or a screenshare or a presentation in full screen, they are having to leave the full screen mode to perform any of the actions such as mute/unmute, camera on/off, answer polls, etc. This results in a bad experience, especially for mute/unmute and polls

**Describe the solution you'd like**
It would be nice if the said action buttons are floated on top of the full screen element, allowing easy access to them.

**Describe alternatives you've considered**
On some devices and browsers, for mute/unmute we are able to use the keyboard shortcuts but that's insufficient and doesn't work on all devices, especially touch only devices.

**Additional context**

**Comment by matiasilva** [comment]
How about keyboard shortcuts? In addition, I believe this is being tackled in the new layout manager in 2.3.

**Comment by mlncn** [comment]
Even more crucial than when full screen is when a person is screensharing.  Having access to the mute and unmute button in particular is crucial.  Participants get confused when BBB is looking at itself and the screen starts recursing to infinity!

@matiasilva Are you proposing keyboard shortcuts as an alternative?  Is there any built-in documentation?  (Or user-facing documentation other than the videos referenced by https://docs.bigbluebutton.org/ ?)

OK, found it, the keyboard shortcuts can be listed from the options menu (three vertical dots in the upper right hand corner).

`Alt + Shift + M`

... not the easiest keyboard combination to reach, especially when primarily using the mouse anyway.

I hope this will be in the new layout manager; is there an issue or area to track this there?

**Comment by gauravgupta7916** [comment]
@ffdixon @matiasilva @antobinary 
Given the mobile-first approach of bigbluebutton (https://docs.bigbluebutton.org/2.2/design.html#mobile-first-approach) the keyboard shortcuts do not seem an appropriate solution here. Would be great if you can guide on the steps or the approach to achieve this and I would be glad to contribute.

**Comment by knarrff** [comment]
Users generally would not know what those shortcuts are and I don't know of universal keybinds for those. Also, the 'infinity-screen' is out-of-scope for this ticket. (Besides: why are people confused?)

The possibility of interaction in full-screen (mute/cam/polls) is absolutely what is expected from a tool like BBB.

**Comment by wolframd** [comment]
+1

**Comment by knarrff** [comment]
As far as I know, this is still an issue...: please don't close, little bot. ;-)

**Comment by knarrff** [comment]
$ ping bot

**Comment by wiebkemueller-hsh** [comment]
We facing the same problems almost daily in meetings and we would very much like to see this implemented, too. Can anything be done budgetwise? Please get in touch @knarrff @ffdixon.

**Comment by Piotr-Fr** [comment]
> @paraviz02 are you aware that there are other options besides sharing the whole screen (share an application or a browser tab)?

Yes, indeed, however there are situations when whole screen is the only solution.
Sharing in BBB needs more dev work, I believe.

**Comment by Piotr-Fr** [comment]
Not sure if I should create another issue but in edu business we have a big need for floating camera window with a user.
Always on top, so no matter what both sides do (client and teacher) they see each other.
With the option to pin (keep on top at all times) and unpin.

**Comment by nidico** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

@stale don't close

**Comment by wiebkemueller-hsh** [comment]
$ ping bot
